uwblueprint/planet-read
# Style login page according to hi-fi designs

## Notion ticket link



[Style login page according to hi-fi designs](https://www.notion.so/uwblueprintexecs/Style-login-page-according-to-hi-fi-designs-5092f235a71e4377a3f3f43e20929695)



## Implementation description

- Added login page styling according to Figma
- Minor colour variation due to Chakra UI defaults and colors.ts file
- Full name input split into first name and last name inputs
- Invalid login doesn't differentiate between wrong email and password (backend not set up for it)
- Forgot password page not built
- No visible indicator for invalid signup (not shown in Figma)



## Steps to test

1. Go to http://localhost:3000/login
2. Enter valid email
3. Enter password
4. Toggle eye icon and check that you can show/hide password
5. Click forgot password and check that alert says reset email was sent
6. Click sign up link and check that page changes
7. Enter first name and last name
8. Enter valid and new email
9. Click register account and check that alert says agree to terms and conditions
10. Check I agree to terms and conditions
11. Click register account and check that you are registered and logged in
12. Make new window and go to http://localhost:3000/login
13. Click sign up link and check that page changes
14. Click login link and check that page changes
15. Test login
16. Test signup with Google
17. Test login with Google



## What should reviewers focus on?

- Is code readable? There is a lot of styling and ternary statements. Please do suggest improvements.
- Does the UI look good (and match the Figma)?
- Does everything still work properly? (Buttons and links especially)

## Checklist
